types/stripe/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for stripe 5.0
+// Type definitions for stripe 6.0.0
```

Comment:
If `listSources` is available in 5.x as well, then I think it would be better to not bump the version in this PR. That way people still on 5.x will get the benefit of the updated definitions ☺️

Reply:
Yeh I wasn't sure if I should change this or not. Will revert.